src/Crypto/WebAuthn/Operation/Registration.hs

```diff
@@ -264,13 +265,33 @@ data RegistrationResult = RegistrationResult
 deriving instance ToJSON RegistrationResult
 
 -- | [(spec)](https://www.w3.org/TR/webauthn-2/#sctn-registering-a-new-credential)
--- The resulting 'rrEntry' of this call should be stored in a database by the
--- Relying Party. The 'rrAttestationStatement' contains the result of the
--- attempted attestation, allowing the Relying Party to reject certain
--- authenticators/attempted entry creations based on policy.
+-- Verifies a 'M.Credential' response for a [registration ceremony](https://www.w3.org/TR/webauthn-2/#registration-ceremony). 
+--
+-- The 'arSignatureCounterResult' field of the result should be inspected to
+-- enforce Relying Party policy regarding potentially cloned authenticators.
```

Comment:
The previous docs shouldn't be removed, and the docs on `arSignatureCounterResult` should be removed as they don't make sense here.

Reply:
copy/paste mistake. thanks!